hunting-buddy.lic

```diff
@@ -128,7 +128,9 @@ class HuntingBuddy
       end
 
       stop_actions(during_actions)
-
+      DRCI.stow_hand('left') unless !DRC.right_hand || !DRC.left_hand
```

Comment:
❓ Why check `right_hand` before stowing the left?

Reply:
Primarily because I've never heard of ;afk causing a problem this way, so I kept the logic unchanged, but the notion of 'stow your left hand unless either your right/left hand is free' makes intuitive sense to me at least.